vassal-doc/src/main/readme-referencemanual/ReferenceManual/GamePiece.adoc

```diff
@@ -198,16 +198,101 @@ Note, however, that Sub-Menu traits must always go *_below_* the traits that it
+
+| <<BasicPiece.adoc#top,Basic Piece>> | Set the Base name and image for a piece.
+
+|<<BorderOutline.adoc#top,Border Outline>>| Specify a colored border around a piece that will appear under specified conditions.
+
+|<<CalculatedProperty.adoc#top,Calculated Property>> | Create a named Property whose value is calculated by a formula.
+
+|<<Pivot.adoc#top,Can Pivot>> | Allows a piece to be rotated around an off-center position.
+
+|<<Rotate.adoc#top,Can Rotate>> | Allows a piece to be rotated around its center.
+
+|<<#Clone,Clone>> | Creates a duplicate of a piece.
+
+|<<#Delete,Delete>> | Deletes the piece from the current game.
+
+|<<NonStacking.adoc#top,Does not stack>> | Provides options to limit stacking, selection and movement of a piece.
+
+|<<Deselect.adoc#top,Deselect>> | Remove a piece from the current piece selection.
+
+| <<DynamicProperty.adoc#top,Dynamic Property>> | Create a named Property that can be changed.
+
+|<<GlobalHotKey.adoc#top,Global Hotkey>> | Issue a <<Concepts.adoc#hotkey,Hotkey>> to activate game <<Concepts.adoc#components,Components>>.
+
+|<<GlobalKeyCommand.adoc#top,Global Key Command>> | Issue a <<Concepts.adoc#keycommand,Key Command>> to activate multiple pieces.
+
+|<<Hideable.adoc#top,Invisible>> | Hide a piece from other players by making it invisible.
+
+|<<Layer.adoc#top,Layer>> | Add additional images to a piece that can turned on and off.
+
+|<<MarkMoved.adoc#top,Mark When Moved>> | Automatically flag a piece whenever it moves.
+
+|<<PropertyMarker.adoc#top,Marker>> | Add a named Property with a fixed value.
+
+|<<Mask.adoc#top,Mask>> | Hide the identity of a piece from other players.
+
+|<<Mat.adoc#top,Mat>> | Create a piece that can carry other pieces around on it.
+
+|<<MatCargo.adoc#top,Mat Cargo>> | Create a pieces that cn be carried on a Mat.
+
+|<<MenuSeparator.adoc#top,Menu Separator>> | Add a separator to the drop-down menu that is generated when you right-click on a piece.
+
+|<<Translate.adoc#top,Move fixed distance>> | Move a piece a set distance.
+
+|<<MovementTrail.adoc#top,Movement Trail>> | Show a visible history of piece movements.
+
+|<<MultiLocationCommand.adoc#top,Multi-location Command>> | Generate multiple menu commands based on Map Zone or Region names.
+
+| <<NonRectangular.adoc#top,Non-Rectangular>> | Specifiy a non-rectangular shape for a piece.
+
+|<<Marker.adoc#top,Place Marker>> | Place a newly created piece.
+
+|<<PlaySound.adoc#top,Play Sound>> | Play a sound.
+
+|<<PropertySheet.adoc#top,Property Sheet>> | Maintain a set of independent values for a piece.
+
+|<<UsePrototype.adoc#top,Prototype>> | Include traits from a pre-defined template piece.
+
+|<<Replace.adoc#top,Replace with Other>> | Replace a piece with a new piece.
+
+|<<ReportChanges.adoc#top,Report Action>> | Display a message in the Chat Window when a Key Command is seen.
+
+|<<RestrictCommands.adoc#top,Restrict Commands>> | Dynamically restrict access to reigh-click menu commands.
```

Comment:
*right-click
Correct me if I'm wrong, but doesn't this trait also stop Named Key Commands from triggering?

Reply:
Yes, it stops that Command being executed at all by GKCs, other traits etc. I've updated the summary.